legend-engine-xt-analytics-store-entitlement-api/src/main/java/org/finos/legend/engine/api/analytics/EntitlementServiceExtension.java

```diff
@@ -0,0 +1,38 @@
+//  Copyright 2023 Goldman Sachs
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package org.finos.legend.engine.api.analytics;
+
+import org.eclipse.collections.api.list.MutableList;
+import org.finos.legend.engine.api.analytics.model.StoreEntitlementAnalyticsInput;
+import org.finos.legend.engine.entitlement.model.entitlementReport.DatasetEntitlementReport;
+import org.finos.legend.engine.entitlement.model.specification.DatasetSpecification;
+import org.finos.legend.engine.language.pure.modelManager.ModelManager;
+import org.pac4j.core.profile.CommonProfile;
+
+import java.util.Collections;
+import java.util.List;
+
+public interface EntitlementServiceExtension
```

Comment:
this looks wrong, this and the service loader should be at the lower layer, i.e. legend-engine-xt-analytics-store-entitlement